daemons/gptp/linux/src/linux_hal_i210.cpp

```diff
@@ -54,50 +54,41 @@ extern "C" {
 static int
 pci_connect( device_t *igb_dev )
 {
-	struct  pci_access      *pacc;
-	struct  pci_dev *dev;
-	int             err;
-	char    devpath[IGB_BIND_NAMESZ];
+	char devpath[IGB_BIND_NAMESZ];
+	struct pci_access *pacc;
+	struct pci_dev *dev;
+	int err;
 
-	memset( igb_dev, 0, sizeof(device_t));
-
-	pacc    =       pci_alloc();
+	memset(igb_dev, 0, sizeof(device_t));
+	pacc = pci_alloc();
 	pci_init(pacc);
 	pci_scan_bus(pacc);
-	for     (dev=pacc->devices;     dev;    dev=dev->next)
-    {
+
+	for (dev = pacc->devices; dev; dev = dev->next) {
 		pci_fill_info(dev, PCI_FILL_IDENT | PCI_FILL_BASES | PCI_FILL_CLASS);
-      
 		igb_dev->pci_vendor_id = dev->vendor_id;
 		igb_dev->pci_device_id = dev->device_id;
 		igb_dev->domain = dev->domain;
 		igb_dev->bus = dev->bus;
 		igb_dev->dev = dev->dev;
 		igb_dev->func = dev->func;
-		
-		snprintf
-			(devpath, IGB_BIND_NAMESZ, "%04x:%02x:%02x.%d", dev->domain,
-			 dev->bus, dev->dev, dev->func );
-
-		err = igb_probe( igb_dev );
-      
+		snprintf(devpath, IGB_BIND_NAMESZ, "%04x:%02x:%02x.%d",
+				dev->domain, dev->bus, dev->dev, dev->func);
+		err = igb_probe(igb_dev);
 		if (err) {
 			continue;
 		}
-
-		printf ("attaching to %s\n", devpath);
-		err = igb_attach( devpath, igb_dev );
-      
+		printf("attaching to %s\n", devpath);
+		err = igb_attach(devpath, igb_dev);
 		if (err) {
-			printf ("attach failed! (%s)\n", strerror(errno));
+			printf("attach failed! (%s)\n", strerror(errno));
 			continue;
 		}
+		/*igb_attach_tx missing here ???*/
```

Comment:
I'm not positive, but I think agb_attach_tx is only needed when dealing with the userspace interface to transmit on the AVB high-priority queues. I don't think the gPTP implementation uses that interface.